packages/angular-cli/commands/version.ts

```diff
@@ -33,15 +40,54 @@ const VersionCommand = Command.extend({
       ngCliVersion = `local (v${pkg.version}, branch: ${gitBranch})`;
     }
 
+    if (projPkg) {
+      roots.forEach(root => {
+        versions = Object.assign(versions, this.getModules(projPkg, root));
+      });
+    }
     this.printVersion('angular-cli', ngCliVersion);
 
     for (const module of Object.keys(versions)) {
-      if (options.verbose || alwaysPrint.indexOf(module) > -1) {
+      let isRoot = false;
+      roots.forEach(root => {
+        if (module.startsWith(root)) {
+          isRoot = true;
+        }
+      });
+      if (options.verbose || alwaysPrint.indexOf(module) > -1 || isRoot) {
         this.printVersion(module, versions[module]);
       }
     }
   },
 
+  getModules: function(pkg: any, prefix: string): any {
+    const modules: any = {};
+    if (pkg.dependencies) {
+      for (let key of Object.keys(pkg.dependencies)) {
+        if (key && key.startsWith(prefix)) {
+          modules[key] = this.getVersion(key);
+        }
+      }
+    }
+    if (pkg.devDependencies) {
+      for (let key of Object.keys(pkg.devDependencies)) {
+        if (key && key.startsWith(prefix)) {
+          modules[key] = this.getVersion(key);
+        }
+      }
+    }
+    return modules;
+  },
+
+  getVersion: function(moduleName: string): string {
+    const modulePkg = require(path.resolve(
```

Comment:
Could you at least try locally to `nom link` angular itself and run `ng version`? Just wanna make sure that works.